e2e_deployment_files/long_haul_deployment.template.windows.json

```
@@ -282,6 +282,29 @@
               "image": "<Container_Registry>/microsoft/azureiotedge-direct-method-receiver:<Build.BuildNumber>-windows-<Architecture>",
               "createOptions": ""
             }
+          },
+          "moduleRestarter": {
+            "version": "1.12",
```

difference from long_haul_deployment.template.json; should it be 1.0?
```suggestion
            "version": "1.0",
```

---

Good catch!